### DISCUSSION TITLE
Create IDataProtectionProvider that stores keys in RavenDB

### DISCUSSION THREAD

**Discussion by marcuslindblom** [ravendb/ravendb#17194] [Q&A] [answered]
I'm curious if someone has created a backing store for data protection using RavenDB?

**Comment by georgiosd** [comment, marked as answer]
Yes, we have done that.

**Reply by marcuslindblom** [reply]
Ok, is it open source or something you would like to share?

**Reply by georgiosd** [reply]
Sorry @marcuslindblom I missed this - do you still need it? I mean, it's fairly easy to make but happy to share.

**Reply by marcuslindblom** [reply]
@georgiosd It would be awesome if you could share it 😊

**Reply by georgiosd** [reply]
Here you go @marcuslindblom, hope it helps

https://gist.github.com/georgiosd/6a3244e334bb355bb9e1e80fae2272da

**Reply by marcuslindblom** [reply]
Awesome, thanks 🙏
